lib/web_ui/dev/driver_manager.dart

```diff
@@ -20,7 +20,34 @@ import 'utils.dart';
 ///
 /// This manager can be used for both macOS and Linux.
 class ChromeDriverManager extends DriverManager {
-  ChromeDriverManager(String browser) : super(browser);
+
+  /// Directory which contains the preferred version of Chrome.
+  ///
+  /// On LUCI we are using the CIPD packages to control Chrome binaries we use.
+  /// There will be multiple CIPD packages loaded at the same time. Yaml file
+  /// `driver_version.yaml` contains the version number we want to use.
+  ///
+  /// Local integration tests are still using the system Chrome. This should be
+  /// fixed in issue: https://github.com/flutter/flutter/issues/53179.
+  ///
+  /// Initialized to the current first to avoid the `Non-nullable` error.
+  io.Directory _browserDriverDirWithVersion = io.Directory.current;
+
+  ChromeDriverManager(String browser) : super(browser) {
+    final io.File lockFile = io.File(pathlib.join(
+        environment.webUiRootDir.path, 'dev', 'driver_version.yaml'));
+    YamlMap _configuration = loadYaml(lockFile.readAsStringSync()) as YamlMap;
+    final String preferredChromeDriverVersion =
+        _configuration['preferred_version']['chrome'] as String;
+    print('preferred_version $preferredChromeDriverVersion');
```

Comment:
This will be popping up during day-to-day usage of `felt`. Consider making the message understandable to someone who doesn't know the format of the `driver_version.yaml` file.

Reply:
done.